docs/guides/hub_guide.md

```diff
@@ -1,15 +1,15 @@
-# IOTA Hub migration guide
+# IOTA Hub Migration Guide
 
-> This guide will explain how you can switch from IOTA Hub to using wallet.rs or its bindings instead.
+> This guide outlined how to switch from IOTA Hub to using wallet.rs or its bindings instead.
 
-Because of the unique features of IOTA 1.0, it was difficult to manage IOTA transactions with just a library. With the new Chrysalis update, the whole protocol was updated to be more accommodating to industry wide standards and developer friendliness. 
+Because of the unique features of IOTA 1.0, it was difficult to manage IOTA transactions with just a library. With the Chrysalis update, the whole protocol was updated to be more accommodating to industry wide standards and developer friendliness. 
```

Comment:
```suggestion
Because of the unique features of IOTA 1.0, it was difficult to manage IOTA transactions with just a library. With the Chrysalis update, we have updated the whole protocol to be more accommodating to industry wide standards and developers.
```